src/styles/main.scss

```
@@ -1,3 +1,68 @@
+@import 'variables';
+
 body {
   margin: 0;
 }
+
+.body {
+  display: flex;
+  justify-content: center;
+}
+
+.calendar {
+  padding: 10px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  min-width: $month-block-width;
+  max-width: $month-block-width;
```

you can go with just setting width:

```suggestion
  width: $month-block-width;
```